upgrade/sql/1.7.8.0.sql

```diff
@@ -159,6 +159,129 @@ ALTER TABLE `PREFIX_log`
 ALTER TABLE `PREFIX_tab` ADD `wording` VARCHAR(255) DEFAULT NULL AFTER `icon`;
 ALTER TABLE `PREFIX_tab` ADD `wording_domain` VARCHAR(255) DEFAULT NULL AFTER `wording`;
 
+UPDATE `PREFIX_tab` SET `wording`='Dashboard', `wording_domain`='Admin.Navigation.Menu' WHERE `class_name`='AdminDashboard';
```

Comment:
I'd say to run the changes in 1.7.8.3, it's highly unlikely a tab will have been reworded by hand in `tab`. Also, you can add this to every line to make extra sure that it hasn't been set already:
```suggestion
UPDATE `PREFIX_tab` SET `wording`='Dashboard', `wording_domain`='Admin.Navigation.Menu' WHERE `class_name`='AdminDashboard' AND COALESCE(`wording`, '') = '' AND COALESCE(`wording_domain`, '') = '';
```

Reply:
Will 1.7.8.3 be released in the near future?

Reply:
https://github.com/PrestaShop/PrestaShop/issues/26969

Reply:
ok, not so far away then :)
Well it sounds that you like the updates moved to 1.7.8.3 then? should I do that change?

Reply:
Yes please!

Reply:
What's the aim of using `COALESCE` with a single value? `COALESCE` is a shortcut for `CASE WHEN`, it returns the first non-null value :thinking: 
Using `wording = '' AND wording_domain = ''` is enough.

Reply:
After checking the `Entity\Tab.php` file, the field is nullable by default, and we authorize empty value, so `IS NULL` should be enough

Reply:
removed the columns from the table?

Reply:
No it's not enough, because both `wording` and `wording_domain` are varchar but default to NULL. So "empty" can be either NULL or empty string. The coalesce works as a shorthand to `(wording = '' OR wording IS NULL)`. 

If we don't coalesce to string and you use IS NULL then we risk leaving out tabs that may have been edited by hand by a developer exploring the original issue, then emptied out to prepare for the update (but not set to NULL). 

I can't see any scenario where you'd like the domain and wording of these tabs empty on purpose.

Reply:
There is a difference between null and non-null values.
> I can't see any scenario where you'd like the domain and wording of these tabs empty on purpose.

Even if you can't see a scenario, I see a lot of them where you can remove unneeded wording or wording_domain and leave fields empty because you don't care about translations in your BO.
If the field mustn't be empty, we have to add a non-empty constraint directly in the BO too and remove the nullable field :thinking:

Reply:
I honestly think your use case is purely hypothetical. I can't imagine a merchant calling their tech guy and say "hey can you please make it so that I can no longer translate the menu items?"

Reply:
I follow what has been done in the code, nothing more :/
We have to be consistent between what we created with `Tab` entity and this one.

Reply:
I understand but we don't control what people might have put in their database. This accounts for that while leaving customization as is.